components/esp-tls/esp_tls.c

```diff
@@ -90,6 +92,8 @@ static const char *TAG = "esp-tls";
 #define _esp_tls_conn_delete                esp_wolfssl_conn_delete
 #define _esp_tls_net_init                   esp_wolfssl_net_init
 #define _esp_tls_server_session_create      esp_wolfssl_server_session_create
+#define _esp_tls_server_session_init        esp_wolfssl_server_session_init
```

Comment:
Sorry for the misunderstanding, but can you please clarify what your suggestion here is?

Reply:
I meant that the support for wolfSSL need not be added. Since I am not sure if you have tested the wolfSSL support yourself and we dont require wolfSSL support to be added so we can just mark the `_esp_tls_server_session_create` as `NULL`.

and in the `esp_tls_server_session_create` API where you use `_esp_tls_server_session_create`
you can just handle this case by doing something like
```
if (_esp_tls_server_session_create == NULL) {
    return ESP_ERR_NOT_SUPPORTED.
}
```